extensions/objects/models/objects.py

```diff
@@ -1196,6 +1196,22 @@ class PositionOfTerms(BaseObject):
     }
 
 
+class RatioExpression(BaseObject):
+    """Class for ratio expression. Stores a unicode string representing a
+    valid ratio expression.
+    """
+
+    description = 'A string for ratio expression.'
+    default_value = '1:1'
+
+    SCHEMA = {
+        'type': 'unicode',
```

Comment:
We shouldn't just be using a string to represent ratios, we should be using a structure like we do for fractions.

Reply:
done